readme.txt

```diff
@@ -1,6 +1,6 @@
 === Advanced Ads for WPBakery Page Builder ===
 Contributors: webzunft, advancedads
-Tags: ads, AdSense, frontend editor, page builder, WPBakery
+Tags: ads, AdSense, page builder, WPBakery, banner, banners, addon, amazon, Google AdSense, drag and drop
```

Comment:
The additional tags seem too generic for this special use case. How about variants of "WPBakery". e.g., "WP Bakery", and also their previous name "Visual Composer"?

Reply:
I added "WP Bakery" and edited the text to contain this (wrong) queue. WPBakery asked us to remove the "visual composer" to avoid confusion with their users. If we only add this tag, we would also need to integrate it into the description that I would avoid.

Reply:
I cannot remember exactly what was behind "visual composer". I thought this was the old name. If that is in fact another product then yes, we should not use it.

Reply:
The Visual Composer became the [WPbakery page builder](https://wpbakery.com/), with which our add-on works. Furthermore, there is now the new free [Visual Composer](https://wordpress.org/plugins/visualcomposer/), with which our add-on does not (yet ) work.